src/modules/joinLeaveNotification.js

```diff
@@ -0,0 +1,56 @@
+const Eris = require("eris");
+const config = require("../config");
+const threads = require("../data/threads");
+const utils = require("../utils/utils");
+const {internalClose} = require("../utils/components");
+
+/**
+ * @param {Eris.CommandClient} bot
+ */
+module.exports = bot => {
+
+    // Join Notification: Post a message in the thread if the user joins main server
+    bot.on("guildMemberAdd", async (guild, member) => {
+        if (guild.id !== (await utils.getMainGuild().id)) return;
+
+        // Attempt to limit database calls by checking if there are existing modmail channels.
+        /**
+        * @type {Eris.CategoryChannel}
+        */
+        const cat = await guild.channels.get(config.modmailCategories[modmail].id);
```

Comment:
This doesn't consider threads that have been moved to other categories. Also `modmail` isn't defined

Reply:
```suggestion
        const config = require("../../config2.json");
        const cat = await guild.channels.get(config.modmailCategories.modmail.id);
```

It's not suppose to consider moved threads, at least not at this point; only main threads as an attempt to limit bulk db calls.